prov/efa/src/efa_ep.c

```diff
@@ -93,11 +93,14 @@ static int efa_ep_modify_qp_state(struct efa_qp *qp, enum ibv_qp_state qp_state,
 	if (attr_mask & IBV_QP_QKEY)
 		attr.qkey = qp->qkey;
 
+	if (attr_mask & IBV_QP_RNR_RETRY)
+		attr.rnr_retry = 0;
```

Comment:
See my internal email about this. Let's put a comment here about what the value means once we hear back in case someone outside Amazon looks at this.